spec/json_column_spec.cr

```diff
@@ -0,0 +1,62 @@
+require "./spec_helper.cr"
+
+class BlobQuery < Blob::BaseQuery
+end
+
+class BlobForm < Blob::BaseForm
+  fillable :doc
```

Comment:
This will need to be `permit_columns` once rebased. And the `BaseForm` is now `SaveOperation`. Maybe it should be `BaseOperation` though? Anyway we can talk about that since it'll be awhile before this big release comes out I think :D

Reply:
`BaseOperation` sounds cool like it's all secret military or something. Do you want me to merge master in to this to make sure it's all working with latest stuff first? Or just wait until after this is merged to update?

Reply:
I think rebasing master would be good to make sure it all works ok. And yeah `BaseOperation` does sound kinda cool haha